adrs/0153-checkout-v2.md

```diff
@@ -179,6 +182,15 @@ A better solution is:
 
 Given a source file path, walk up the directories until the first `.git/config` is found. Check if it matches the self repo (`url = https://github.com/OWNER/REPO`). If not, drop the source file path.
 
+### Submodules
```

Comment:
This section assumes `fetch-refs` and SSH key support will be added - currently in separate PR proposals.

Reply:
I think only fetch depth applies. Will yank this part about fetch refs (not implemented yet) and revisit in ADR for fetch-refs